packaging/windows/variables.wxi

```diff
@@ -26,7 +26,7 @@
   <?if $(var.Platform)=x86?>
   <?define Program_Files="ProgramFilesFolder"?>
   <?define Win64AttributeValue=no?>
-  <?define UpgradeCode="70A1576F-63B6-4659-9E39-25C7B769DDE5"?>
+  <?define UpgradeCode="25CAE344-22A6-422D-942E-942687C56A45"?>
```

Comment:
This is the `UpgradeCode` which used for product identification by installer services. Mainly for deciding whether it is a fresh install or an upgrade over a previous install. 

This is the first time we are generating x86 installers, so I wanted to start off with a fresh `UpgradeCode`. I do not remember where I got the previous upgrade code from.